### ISSUE TITLE
Cap request lengths

### ISSUE THREAD

**Issue by RichardTaylor** [mysociety/alaveteli#5979]
Misuse of the service is often associated with excessively long requests. 

Perhaps ideally there would be a cap on request length, perhaps one which could be varied by administrators on a user by user basis. 

Related:

Message to users considering making very long follow ups   #3387 

Warn user when their new request is too wordy https://github.com/mysociety/alaveteli/pull/4987

**Comment by garethrees** [comment]
We already warn when messages are getting long. I don't think a hard cap will really help if someone is intentionally trying to misuse, as they'll find a way around it, and will get in the way of users with genuine reason to write longer messages. Better to target the specific attributes of misuse.
